brayns/common/scene/Scene.cpp

```diff
@@ -247,78 +235,78 @@ void Scene::buildDefault()
                                 {0.8f, 0.8f, 0.8f}, {0.f, 1.f, 0.f},
                                 {0.8f, 0.8f, 0.8f}, {0.8f, 0.8f, 0.8f}};
 
-    // Cornell box
-    for (size_t material = 1; material < 6; ++material)
+    for (size_t i = 1; i < 6; ++i)
     {
-        auto& m = _materials[NB_SYSTEM_MATERIALS + material];
-        m.setColor(colors[material]);
-        m.setSpecularColor(WHITE);
-        m.setSpecularExponent(10.f);
-        m.setReflectionIndex(material == 4 ? 0.8f : 0.f);
-        m.setOpacity(1.f);
-        auto& trianglesMesh = _trianglesMeshes[NB_SYSTEM_MATERIALS + material];
-        for (size_t i = 0; i < 6; ++i)
+        // Cornell box
+        Material material;
+        material.setColor(colors[i]);
```

Comment:
constructor for Material to take some/all of this values?

Reply:
hum... it's either all or none, some would be a bit weird to me. And having them all makes it a long list of arguments. Maybe another idea would to be to define some presets, like Metal, Glass, etc?